forge-gui/res/cardsfolder/upcoming/call_to_the_void.txt

```diff
@@ -0,0 +1,6 @@
+Name:Call to the Void
+ManaCost:4 B
+Types:Sorcery
+A:SP$ ChooseCard | Defined$ Player | Choices$ Creature | SecretlyChoose$ True | Amount$ 1 | ControlAndNot$ True | ChoiceTitle$ Secretly choose a creature | Reveal$ True | RevealTitle$ OVERRIDE Chosen creatures. They will be destroyed. | SubAbility$ DBDestroyChosen | RememberChosen$ True | Mandatory$ True
+SVar:DBDestroyChosen:DB$ DestroyAll | ValidCards$ Creature.IsRemembered | SubAbility$ DBCleanUp
```

Comment:
You might not need IsRemembered and just use ChosenCard instead

Reply:
I agree!